scripts/images.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+IMAGES=(
+  # These must be built first because others build FROM these
```

Comment:
Hard-coded these instead of doing a fancy list-image thing, just makes it easier for us to control the ordering (if we want to do so)